images_config/ks-robot-slave.cfg

```diff
@@ -280,77 +172,106 @@ sed -i 's/NEED_STADT\=/NEED_STADT\=yes/g' /etc/default/nfs-common
 update-rc.d autofs defaults
 service autofs restart
 modprobe nfs
+{ rm /etc/default/nfs-common && awk '/^NEED_STATD=/ {sub(/\=/,"=yes")}1' > /etc/default/nfs-common; } < /etc/default/nfs-common
```

Comment:
I have tried many ways to make NEED_STATD=yes which was on NEED_STATD in the file of /etc/default/nfs-common , its more customized format which removes and replace same file with required changes